packages/react/src/components/Table/TableBody/TableBodyLoadMoreRow/TableBodyLoadMoreRow.test.jsx

```diff
@@ -2,23 +2,55 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
+import { settings } from '../../../../constants/Settings';
+
 import TableBodyLoadMoreRow from './TableBodyLoadMoreRow';
 
-const tableRowProps = {
+const { prefix } = settings;
+
+const getTableRowPropsNotLoading = () => ({
   id: 'tableRow01',
   tableId: 'tableId',
   testId: 'table-test',
   totalColumns: 5,
   loadMoreText: 'Load more...',
   isLoadingMore: false,
-};
+  rowVisibilityRef: null,
+  hasRowActions: true,
+  hasRowExpansion: false,
+  hasRowNesting: true,
+  hasRowSelection: false,
+  showExpanderColumn: false,
+  columns: [
+    {
+      id: 'string',
+      name: 'String',
+      isSortable: true,
+    },
+    {
+      id: 'select',
+      name: 'Select',
+      isSortable: true,
+    },
+    {
+      id: 'number',
+      name: 'Number',
+      isSortable: true,
+    },
+  ],
+});
+
+const getTableRowPropsLoading = () => ({
+  ...getTableRowPropsNotLoading(),
+  isLoadingMore: true,
+});
```

Comment:
and then using the `overrides` comment above, you can simply call `getTableRowProps({isLoadingMore: true})` to get the Loading version instead of calling two functions.